portality/bll/background_task_status.py

```diff
@@ -0,0 +1,138 @@
+"""
```

Comment:
This file should be in `portality.bll.services`

Reply:
Also, if it's going in as a core service, it needs to follow the same structure, so to be an object with methods.